eras/babbage/formal-spec/rewards.tex

```diff
@@ -0,0 +1,51 @@
+\newcommand{\Stake}{\type{Stake}}
+\newcommand{\mReward}[4]{\fun{r_{member}}~ \var{#1}~ \var{#2}~ \var{#3}~ {#4}}
+\newcommand{\lReward}[4]{\fun{r_{operator}}~ \var{#1}~ \var{#2}~ \var{#3}~ {#4}}
+
+\section{Forgo Reward Calculation Prefilter}
+
+The reward calculation no longer filters out the unregistered stake
+credentials when creating a reward update. As in the Shelley era, though,
+they are still filtered on the epoch boundary when the reward update is applied.
+This addresses errata 17.2 in the Shelley ledger specification \cite{shelley_spec}[17.2].
+The change consists of removing the line
+$$\var{addrs_{rew}}\restrictdom{\var{potentialRewards}}$$
+from the $\fun{rewardOnePool}$ function.
+
+\begin{figure}[htb]
```

Comment:
It seems a real shame to have to add this big table for such a simple edit (removing one short line and renaming one variable). Perhaps we can do better? Just describe the diff? :thinking:

Reply:
Yeah, I'd put three dots for everything in the where block except the last line, and maybe highlight that one

Reply:
I like that much better, thanks!